lexxpluss_apps/src/can_main.cpp

```diff
@@ -66,10 +57,14 @@ class {
     ros::NodeHandle nh;
     ros_bmu bmu;
     ros_board board;
+<<<<<<< HEAD:lexxpluss_apps/src/can_main.cpp
+    ros_imu imu;
+=======
     ros_dfu dfu;
     can_imu imu;
     ros_interlock interlock;
     ros_tof tof;
+>>>>>>> 9fb6710ba46d8b4e87798329b69dd978f79fe102:lexxpluss_apps/src/rosserial.cpp
```

Comment:
@shujima 
この辺りが原因でビルドに失敗している。
The build is failing because of this changes.

https://github.com/LexxPluss/LexxHard-SensorControlBoard-Firmware/actions/runs/8386100342/job/22965952859?pr=12#step:4:1316

Reply:
ビルドが失敗する問題は解消しました。

Reply:
ありがとうございます！
Thank you very much!